omero/sysadmins/public.txt

```diff
@@ -0,0 +1,135 @@
+Public data in the repository
+=============================
+
+The OMERO.web framework allows raw data to be published using built-in tools
+or supplied through webservices to the external web resources. This feature is
+available only after granting access to the selected datasets and making them
+visible to the 'public user', ensuring you always have control over access to
+your data.
+
+There are three ways of publishing data using OMERO.web:
+
+-   using a URL to launch the web-based Image viewer, as described in
+    :ref:`launching_web_viewer`, which can be accompanied by thumbnail. For
+    more details how to load the thumbnail see
+    :ref:`urls_from_within_OMERO_web`.
```

Comment:
I don't think we end each list item with a "." if they are not a sentence starting with an initial capital.